src/main/java/com/amazon/opendistroforelasticsearch/ad/model/AnomalyDetectorType.java

```diff
@@ -27,8 +27,9 @@
 package com.amazon.opendistroforelasticsearch.ad.model;
 
 public enum AnomalyDetectorType {
-    REALTIME_SINGLE_ENTITY,
-    REALTIME_MULTI_ENTITY,
+    SINGLE_ENTITY,
```

Comment:
Can we change single entity to single stream and multi entity to HCAD?  Discussed with science team and we use the same term in blogs. This applies to other similar places.

Reply:
Prefer to create a separate PR to rename all places to keep it easier to review. Create an [issue](https://github.com/opensearch-project/anomaly-detection/issues/44) to track this.